WeightProducer/python/MCSampleValues.py

```diff
@@ -63,493 +63,1039 @@ class MCSampleValuesHelper():
+                XS_2017=8640.0, XSSource_2017="XSDB (LO)",
+                XS_2018=8608.0, XSSource_2018="GenXSecAnalyzer",
+            ),
         },
         "GJets_HT-200To400" : {
-            "CrossSection" : XSValues(XS_13TeV=2300.0),
+            "CrossSection" : XSValues(
+                XS_2016=2321.0, XSSource_2016="XSDB (LO)",
+                XS_2017=2185.0, XSSource_2017="XSDB (LO)",
+                XS_2018=2190.0, XSSource_2018="GenXSecAnalyzer",
+            ),
         },
         "GJets_HT-400To600" : {
-            "CrossSection" : XSValues(XS_13TeV=277.4),
+            "CrossSection" : XSValues(
+                XS_2016=275.2, XSSource_2016="XSDB (LO)",
+                XS_2017=258.5, XSSource_2017="GenXSecAnalyzer",
+                XS_2018=258.0, XSSource_2018="GenXSecAnalyzer",
+            ),
         },
         "GJets_HT-600ToInf" : {
-            "CrossSection" : XSValues(XS_13TeV=93.38),
+            "CrossSection" : XSValues(
+                XS_2016=93.19, XSSource_2016="XSDB (LO)",
+                XS_2017=85.31, XSSource_2017="XSDB (LO)",
+                XS_2018=85.11, XSSource_2018="GenXSecAnalyzer",
+            ),
         },
         "GJets_DR-0p4_HT-100To200" : {
-            "CrossSection" : XSValues(XS_13TeV=5000.0),
+            "CrossSection" : XSValues(
+                XS_2016=5363.0, XSSource_2016="XSDB (LO)",
+                XS_2017=5044.0, XSSource_2017="GenXSecAnalyzer",
+                XS_2018=5030.0, XSSource_2018="GenXSecAnalyzer",
+            ),
         },
         "GJets_DR-0p4_HT-200To400" : {
-            "CrossSection" : XSValues(XS_13TeV=1079.0),
+            "CrossSection" : XSValues(
+                XS_2016=1178.0, XSSource_2016="XSDB (LO)",
+                XS_2017=1130.0, XSSource_2017="GenXSecAnalyzer",
+                XS_2018=1125.0, XSSource_2018="GenXSecAnalyzer",
+            ),
         },
         "GJets_DR-0p4_HT-400To600" : {
-            "CrossSection" : XSValues(XS_13TeV=125.9),
+            "CrossSection" : XSValues(
+                XS_2016=131.8, XSSource_2016="XSDB (LO)",
+                XS_2017=124.6, XSSource_2017="GenXSecAnalyzer",
+                XS_2018=124.7, XSSource_2018="GenXSecAnalyzer",
+            ),
         },
         "GJets_DR-0p4_HT-600ToInf" : {
-            "CrossSection" : XSValues(XS_13TeV=43.36),
+            "CrossSection" : XSValues(
+                XS_2016=44.27, XSSource_2016="XSDB (LO)",
+                XS_2017=40.65, XSSource_2017="GenXSecAnalyzer",
+                XS_2018=40.46, XSSource_2018="GenXSecAnalyzer",
+            ),
         },
         "ZJetsToNuNu_Zpt-100to200" : {
-            "CrossSection" : XSValues(XS_13TeV=43.5543),
+            "CrossSection" : XSValues(
+                XS_2016=35.99, XSSource_2016="XSDB (LO)",
+            ),
         },
         "ZJetsToNuNu_Zpt-200toInf" : {
-            "CrossSection" : XSValues(XS_13TeV=5.28654),
-        },
-        "ST_s-channel_4f_hadronicDecays" : {
```

Comment:
we seem to have lost this sample

Reply:
I'm putting it back in. It's only available in 2017 and 2018. I have to run the GenXSecAnalyzer over this one.

Reply:
Should be replaced now.